source/views/components/colors.js

```diff
@@ -82,6 +82,7 @@ export const black75Percent = 'rgb(191.25, 191.25, 191.25)'
 export const warmGray = 'rgb(133, 117, 112)'
 export const coolGray = 'rgb(118, 122, 133)'
 export const charcoal = 'rgb(34, 34, 34)'
+export const lightGray = '#8E8E93'
```

Comment:
Why a new color?

Reply:
It's the color of the detail text in the ios cell..
<img width="300" alt="screen shot 2018-02-02 at 1 36 32 pm" src="https://user-images.githubusercontent.com/24640726/35751439-206f8bde-081e-11e8-8b8f-6a7c609cecf3.png">

Reply:
Would you change it to `rgb(142, 142, 147)` (the rgb equivalent), change iosDisabledText to this value, and use that variable instead?